.github/workflows/ci-build.yml

```diff
@@ -56,26 +56,56 @@ on:
         required: false
         type: boolean
         default: true
+      REGISTRY:
+        type: string
+        required: false
+        default: registry.hub.docker.com
+      TOKEN_FORMAT:
+        type: string
+        default: id_token
+        description: 'The default token format generated when authenticating.'
+      IMAGE_REPO:
+        type: string
+        required: false
+        default: 'signalwire'
+        description: 'The default registry repo owner.'
+      CONTAINER_SCAN:
+        type: boolean
+        default: true
+        description: 'Whenever Trivy scan should run.'
+      CONTAINER_TEST:
+        type: boolean
+        default: true
+        description: 'Whenever the container should be tested on pr.'
+      CONTAINER_TEST_COMMAND:
+        type: string
+        default: ''
+        description: 'A custom command to test the docker container.'
     secrets:
       DOCKERHUB_USERNAME:
-        required: true
+        required: false
       DOCKERHUB_TOKEN:
-        required: true
-      GITHUBTOKEN:
-        required: true
+        required: false
       GPG_PASSPHRASE:
         required: false
       GH_BOT_DEPLOY_KEY:
         required: false
+      GCP_SA:
+        required: false
+      GCP_WIP:
+        required: false
 
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
+  
   build:
     # The type of runner that the job will run on
-    runs-on: ubuntu-latest 
+    runs-on: ubuntu-latest-16-cores
```

Comment:
Can we make this configurable? Not every build should require a 16-core runner but if we can specify it for something like FS builds that would be great.